CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Changelog for Pulse3D
 =====================
 
+
+0.30.5 (2023-03-13)
```

Comment:
will these changes be included in todays release?

Reply:
They won't be QCed until the next round of QC, so no. Also we are pushing the release back until tomorrow